ios/demo/src/MapViewController.m

```diff
@@ -32,6 +32,22 @@ - (void)mapView:(TGMapViewController *)mapView didLoadSceneAsync:(NSString *)sce
     [mapView setPosition:newYork];
 }
 
+- (void)mapView:(TGMapViewController *)mapView didSelectMarker:(TGMapMarkerId)markerId atScreenPosition:(CGPoint)position
+{
+    if (markerId == 0) {
```

Comment:
Just curious - would we ever want to call the delegate callback if the markerId is zero? Seems like something I wouldn't expect as an implementing developer.